Makefile

```diff
@@ -100,6 +107,8 @@ clean-hub: clean-hub-cr clean-hub-operator
 
 clean-spoke: clean-spoke-cr clean-spoke-operator
 
+clean-spoke-detached: clean-spoke-cr-detached clean-spoke-operator
```

Comment:
Since now we can deploy several spoke on one cluster, I wonder whether do we have the case that the user only wants to clean a specific spoke on the cluster?

Reply:
Do you mean a user only wants to clean the klusterlet for a specific spoke cluster? I think deleting the `Klusterlet` directly can help.
The target `clean-spoke-detached` in Makefile just covers the case to clean spoke CRs and operator that the `deploy-spoke-detached` target created.

Reply:
Yes, that makes sense.